components/canvas/canvas_paint_thread.rs

```diff
@@ -544,6 +544,8 @@ impl<'a> CanvasPaintThread<'a> {
 
     fn recreate(&mut self, size: Size2D<i32>) {
         self.drawtarget = CanvasPaintThread::create(size);
+        self.state = CanvasPaintState::new(self.state.draw_options.antialias == AntialiasMode::Default);
```

Comment:
Could you file a `E-Easy` follow-up issue about using `AntialiasMode` directly in `CanvasPaintState::new` while at it? `CanvasPaintState` uses `AntialiasMode` too internally, setting the value from the boolean parameter.